richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/ui/BasicRichTextEditor.kt

```diff
@@ -189,6 +188,7 @@ internal fun BasicRichTextEditor(
     maxLength: Int = Int.MAX_VALUE,
     onTextLayout: (TextLayoutResult) -> Unit = {},
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    uriHandler: UriHandler = LocalUriHandler.current,
```

Comment:
For overriding the `UriHandler` we can use CompositionLocal. There's no need to pass it as a parameter.

Reply:
pushed that change